templates/setup/linux/angkor/ubuntu_20.04.sh

```diff
@@ -0,0 +1 @@
+echo pepe
```

Comment:
?

Reply:
This was for testing missing dependencies, it leaked 🤣, will remove it